tests/docker-stacks-foundation/test_python_version.py

```diff
@@ -17,8 +17,9 @@ def test_python_version(container: TrackedContainer) -> None:
         tty=True,
         command=["python", "--version"],
     )
-    assert logs.startswith("Python ")
-    full_version = logs.split()[1]
+    python = logs.splitlines()[7]
```

Comment:
I don't really like hardcoding line number. Any log line added or removed from `start.sh` will break this test for no reason.

To solve this, for example, we can choose the line which starts with `Python ` instead.

The same thing is true for the test below.